prompb/types.proto

```diff
@@ -0,0 +1,39 @@
+
+syntax = "proto3";
+package prometheus;
+
+option go_package = "prompb";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+message Sample {
+  double value    = 1;
+  int64 timestamp = 2;
+}
+
+message TimeSeries {
+  repeated Label labels   = 1;
+  repeated Sample samples = 2;
+}
+
+message Label {
+  bytes name  = 1;
+  bytes value = 2;
+}
+
+message Labels {
+  repeated Label labels = 1 [(gogoproto.nullable) = false];
+}
+
+// Matcher specifies a rule, which can match or set of labels or not.
+message LabelMatcher {
+  enum Type {
```

Comment:
nit: think these are usually defined separately rather than in the `message`?

Reply:
Got rid of this file.